catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -140,7 +139,7 @@
 
     @Before
     public void setUp() throws Exception {
-        framework = createFramework();
+        framework = createFramework(2);
```

Comment:
❗️ I notice this same method is called in your individual tests, so is should this remain? Also the method name is rather vague, leading a reader to wonder what the 2 is there for. Specifying that this sets up a mock would be very helpful.